### ISSUE TITLE
Updating to 3.4.3 results in NoSuchMethodError: The getter 'isInit' was called on null

### ISSUE THREAD

**Issue by nemoryoliver** [jonataslaw/getx#457]
**Describe the bug**
After updating to version 3.4.3 I get this error when running my app. No code changes were made and reverting back to 3.4.2 there's no error.

```
I/flutter ( 2754): NoSuchMethodError: The getter 'isInit' was called on null.
I/flutter ( 2754): Receiver: null
I/flutter ( 2754): Tried calling: isInit
I/flutter ( 2754):
I/flutter ( 2754): #0      Object.noSuchMethod  (dart:core-patch/object_patch.dart:51:5)
I/flutter ( 2754): #1      GetInstance.initDependencies 
package:get/…/instance/get_instance.dart:102
I/flutter ( 2754): #2      GetInstance.find 
package:get/…/instance/get_instance.dart:167
I/flutter ( 2754): #3      GetInstance.put 
package:get/…/instance/get_instance.dart:51
I/flutter ( 2754): #4      Inst.put 
package:get/…/instance/extension_instance.dart:23
I/flutter ( 2754): #5      new PostScreenController 
package:app/…/post/screen.controller.dart:22
I/flutter ( 2754): #6      AppController._initControllers 
package:app/…/app/app.controller.dart:119
I/flutter ( 2754): #7      AppController.onInit 
package:app/…/app/app.controller.dart:89
I/flutter ( 2754): #8      DisposableInterface.onStart 
package:get/…/rx/rx_interface.dart:30
I/flutter ( 2754): #9      GetInstance.startController 
package:get/…/instance/get_instance.dart:128
I/flutter ( 2754): #10     GetInstance.initDependencies 
package:get/…/instance/get_instance.dart:104
I/flutter ( 2754): #11     GetInstance.find 
package:get/…/instance/get_instance.dart:167
I/flutter ( 2754): #12     GetInstance.put (packa
```

**Flutter Version:**
Channel beta, 1.20.0

**Get Version:**
3.4.3

**Describe on which device you found the bug:**
OnePlus 8 - Real device

**Minimal reproduce code**
Basically, I have an **AppController** which is the very parent controller and overrides the onInit and initializes **PostScreenController** using `Get.put`

**Comment by Nipodemos** [comment]
Hello! 
Please, it is better if you provide a minimal reproduce code, only explain the code usually leads to not being able to reproduce your issue locally

**Comment by nemoryoliver** [comment]
> Hello!
> Please, it is better if you provide a minimal reproduce code, only explain the code usually leads to not being able to reproduce your issue locally

Will do in a bit... thanks

**Comment by nemoryoliver** [comment]
Here's a minimal reproducible code

```
import 'package:flutter/material.dart';
import 'package:get/get.dart';

void main() {
  runApp(MyApp());
}

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    Get.put(AppController());
    return GetMaterialApp(home: HomeScreen());
  }
}

class HomeScreen extends StatelessWidget {
  const HomeScreen({Key key}) : super(key: key);

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(title: Text('Test')),
      body: Text('Test'),
    );
  }
}

class AppController extends GetxController {
  static AppController get to => Get.find();

  @override
  void onInit() async {
    Get.put(PostScreenController());
    super.onInit();
  }
}

class PostScreenController extends GetxController {
  final friendsController =
      Get.put(FriendsController(pickerMode: true), tag: 'sheet');
}

class FriendsController extends GetxController {
  final bool pickerMode;
  FriendsController({this.pickerMode = false});
}
```

**Comment by nemoryoliver** [comment]
It looks like it has something to do with the **tag** set to the **FriendsController**. Removing the tag fixes the error

**Comment by jonataslaw** [comment]
This has already been fixed by https://github.com/jonataslaw/getx/commit/d208a613833948b44c1839175b084d40ca29dae1

I will update pub.dev soon.

**Comment by jonataslaw** [comment]
fix com 3.4.4

**Comment by nemoryoliver** [comment]
Great to hear! Thanks a lot
